pyntc/devices/aireos_device.py

```diff
@@ -63,7 +63,7 @@ class AIREOSDevice(BaseDevice):
 
     vendor = "cisco"
 
-    def __init__(self, host, username, password, secret="", port=22, **kwargs):
+    def __init__(self, host, username, password, secret="", port=22, **kwargs):  # nosec
```

Comment:
what is the security issue being reported?

Reply:
I think this is B107, which we should just ignore in the bandit config file

Reply:
That is correct. Added B107 to exclude list on bandit config file.

Reply:
Sorry, I meant that when adding that to the exclude list, you would remove the `# nosec` comments from the corresponding function defs

Reply:
I did that. The function with the `nosec` still is bandit test B501.